doc/U-19_Initialize_game.md

```diff
@@ -0,0 +1,21 @@
+# U-19: Initialize game
+**Actor:** Admin
+
+**Preconditions:** -
+
+**Postconditions:**
+
+ - Team status for every cypher is reset (first is in status pending)
```

Comment:
Just to be on the safe side: `Team status for every cypher is reset (first cypher is in status pending)`

Reply:
ok